codemod_tox/parse.py

```diff
@@ -1,6 +1,9 @@
 import re
 
 TOX_ENV_TOKEN_RE = re.compile(
-    r"(?P<comma>,[ \t]*)|(?P<newline>\n)|(?P<space>[ \t]+)|(?P<literal>[A-Za-z0-9-_]+)|(?P<options>\{[^{}\s]+\})"
+    r"(?P<comma>,[ \t]*)|(?P<newline>\n)|(?P<space>[ \t]+)|(?P<literal>[A-Za-z0-9_-]+)|(?P<options>\{[^{}]+\})"
```

Comment:
note the negated character class no longer includes `[^...\s]` because it turns out whitespace is allowed inside options :/

Still disallows close curly so as long as these things can't be quoted or escaped, I think it's ok.